### ISSUE TITLE
Package stats doesn't work in 5.23.0

### ISSUE THREAD

**Issue by rtoy** [rtoy/maxima#1808]
Imported from SourceForge on 2024-07-05 08:43:35
Created by **[blahota](https://sourceforge.net/u/blahota/)** on 2011-01-11 07:59:11
Original: https://sourceforge.net/p/maxima/bugs/2135

---

I wanted to use simple statistics, like "test\_mean", and I didn't get real result. I also tried another stats functions, they don't work.
\(For example 'mean'.\) See attached wxmx \(wxMaxima\) file.

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-05 08:43:40
Created by **[riotorto](https://sourceforge.net/u/riotorto/)** on 2011-01-11 11:17:36
Original: https://sourceforge.net/p/maxima/bugs/2135/#2550

---

- **status**: open --> closed

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-05 08:43:44
Created by **[riotorto](https://sourceforge.net/u/riotorto/)** on 2011-01-11 11:17:36
Original: https://sourceforge.net/p/maxima/bugs/2135/#e86e

---

Blahota,

There are two problems here:

1.- The error referring to file "grcommon.lisp" was already fixed in cvs. This error prevents loading package 'stats', that's why you don't get any results.

2.- The warning on MATHML-LIST \(returned by GCL and SBCL\) is motivated by some code I introduced time ago but is no longer necessary. I have removed it.

So, I think everything is now ok.

Thanks for the bug report.

\--
Mario
